utils/core-contracts/contracts/token/ERC20PermitStorage.sol

```diff
@@ -0,0 +1,17 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+library ERC20PermitStorage {
+    struct Data {
+        uint256 initialChainId;
+        bytes32 initialDomainSeprator;
+        mapping(address => uint256) nonces;
+    }
+
+    function load() internal pure returns (Data storage store) {
+        bytes32 s = keccak256(abi.encode("ERC20PermitStorage"));
```

Comment:
Use constant, and "io.synthetix.core-contracts.ERC20Permit". See https://github.com/Synthetixio/synthetix-v3/pull/1295/files#diff-5c38ca900eb44a689325c649eed3159283a48c9281223c8d33967a658e60d796R11

Reply:
Might need that PR merged first so that tooling can deal with constants in storage dumps.